homeassistant/components/home_connect/select.py

```diff
@@ -268,18 +275,20 @@ async def async_select_option(self, option: str) -> None:
         """Select new program."""
         bsh_key = TRANSLATION_KEYS_PROGRAMS_MAP[option]
         _LOGGER.debug(
-            "Tried to start program: %s"
-            if self.start_on_select
-            else "Tried to select program: %s",
+            "Starting program: %s" if self.start_on_select else "Selecting program: %s",
             bsh_key,
         )
         try:
-            await self.hass.async_add_executor_job(
-                self.device.appliance.start_program
-                if self.start_on_select
-                else self.device.appliance.select_program,
-                bsh_key,
-            )
+            if self.start_on_select:
+                await self.hass.async_add_executor_job(
```

Comment:
Store the parameters in a tuple, local variable in the if/else block. Then pass that variable, unpacking it, once to `hass.async_add_executor_job`.